mungegithub/mungers/assign-fixes.go

```diff
@@ -47,6 +59,15 @@ func (a *AssignFixesMunger) RequiredFeatures() []string { return []string{} }
 func (a *AssignFixesMunger) Initialize(config *github.Config, features *features.Features) error {
 	a.features = features
 	a.config = config
+	a.collaborators = sets.String{}
```

Comment:
I'd think you'd want to get the collaborators EachLoop instead of once per run. It could become pretty out of date if we ever let the bot run a while.